src/Bonnier/WP/Cache/Models/Post.php

```diff
@@ -8,37 +8,106 @@
 class Post
 {
     private static $settings;
+    // Figure out a way to dynamically fetch this variable - otherwise
+    // REMEMBER to update this field whenever ACF field keys are updated
+    // if they ever are.
+    const ACF_CATEGORY_ID = 'field_58e39a7118284';
 
     public static function watch_post_changes(SettingsPage $settingsPage)
     {
         self::$settings = $settingsPage;
 
-        add_action('transition_post_status', [__CLASS__, 'post_status_changed'], 10, 3);
+        // triggers when post status changes to trash
+        add_action('wp_trash_post', [__CLASS__, 'remove_post'], 10, 1);
+        add_action('publish_to_draft', [__CLASS__, 'remove_post'], 10, 1);
+
+        // publish post
+        add_action('publish_to_publish', [__CLASS__, 'update_posts'], 10, 1);
```

Comment:
can you rename the function name to be similar to the remove & publish .. "singular"

Reply:
yes sir!